ThumbnailItem/ThumbnailItem.module.less

```diff
@@ -44,23 +45,7 @@
 
```

Comment:
On line ~39 let's add 
`.slotBefore {
	margin-inline-end: 21px;
}` (create a silicon variable). To meet GUI requirements
![margin-right](https://user-images.githubusercontent.com/63335823/114154441-15009280-9929-11eb-916a-dfc8c3de12d9.PNG)